developer-community/events-list.md

```diff
@@ -1,3 +1,5 @@
+(DEPRECATED: please follow the instructions in github.com/src-d/conferences to track new events)
```

Comment:
You could format the link as follows `[src-d/conferences](https://github.com/src-d/conferences)` or similar.